src/library-authoring/LibraryLayout.tsx

```diff
@@ -25,7 +26,16 @@ const LibraryLayout = () => {
   }
 
   return (
-    <LibraryProvider key={collectionId} libraryId={libraryId} collectionId={collectionId}>
+    <LibraryProvider
+      key={collectionId}
+      libraryId={libraryId}
+      collectionId={collectionId}
+      /** The component picker modal to use. We need to pass it as a reference instead of
+       * directly importing it to avoid the import cycle:
+       * ComponentPickerModal > ComponentPicker > LibraryAuthoringPage/LibraryCollectionPage >
+       * Sidebar > AddContentContainer > ComponentPickerModal */
```

Comment:
```suggestion
      {/** The component picker modal to use. We need to pass it as a reference instead of
        * directly importing it to avoid the import cycle:=
        * ComponentPickerModal > ComponentPicker > LibraryAuthoringPage/LibraryCollectionPage >
        * Sidebar > AddContentContainer > ComponentPickerModal */}
```
My jsx is rusty, but doesn't this need to be in curly braces?

Reply:
Apparently it's not necessary.